### PR TITLE
pmset: add page

### LINKED ISSUE #952: add pmset command (OS X)
not found this command's man page on mac

### PR DESCRIPTION
**Pull request by jonboiser**
Fixes #952.

Took these examples from the man page, based on uses that I personally found useful and interesting. Let me know if you have any feedback.

### CODE REVIEW

File: pages/osx/pmset.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# pmset
+
+> Configure macOS power management settings, as one might do in System Preferences > Energy Saver.
+> These commands must be run as root, i.e. starting with `sudo`.
+
+- See the current power management settings:
```
Comment (agnivade) [inline pages/osx/pmset.md]: Would be good to change from "See" to "Display".

File: pages/osx/pmset.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`pmset -g`
+
+- See current power source and battery levels:
```
Comment (agnivade) [inline pages/osx/pmset.md]: same as above.

File: pages/osx/pmset.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Set display to never sleep when on charger power:
+
+`pmset -c displaysleep 0`
+
+- Have display sleep after 15 minutes when on battery power:
+
+`pmset -b displaysleep 15`
+
+- Restore to system defaults:
```
Comment (agnivade) [inline pages/osx/pmset.md]: I would move this command to the end as this seems to be the most complex one.

Comment (jonboiser) [inline pages/osx/pmset.md]: made changes and did a force update

Comment (agnivade) [inline pages/osx/pmset.md]: Thanks, you need not had done the force update btw :). Incremental commits are totally fine and help understand the changes done.

**Comment by agnivade** [comment]
LGTM

File: pages/osx/pmset.md (hunk 1 above)
Comment (waldyrious) [inline pages/osx/pmset.md]: or "Show".

**Comment by waldyrious** [comment]
We've agreed previously that commands that require sudo should include it directly in the examples, for convenience -- see e.g. [nethogs](../blob/master/pages/linux/nethogs.md), [route](../blob/master/pages/osx/route.md), [iptables](../blob/master/pages/linux/iptables.md).

**Comment by jonboiser** [comment]
Added sudo to the relevant examples. Should I get rid of that second line in the description re: using sudo?

**Comment by waldyrious** [comment]
Yeah, I suppose the second line can be removed, or instead clarify when it is needed, like you did in the commit message :)

**Comment by jonboiser** [comment]
I simplified the description, explaining when `sudo` is required.

**Comment by waldyrious** [comment]
Thanks, LGTM 👍

**Pull request merged**